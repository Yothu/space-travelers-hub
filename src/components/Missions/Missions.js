const Missions = () => (
  <div>
    <div>
      <p>
        missions
      </p>
    </div>
  </div>
);

export default Missions;
