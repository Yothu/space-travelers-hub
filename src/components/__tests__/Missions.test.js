import { Provider } from 'react-redux';
import renderer from 'react-test-renderer';
import Missions from '../Missions/Missions';
import store from '../../redux/configureStore';

describe('Missions page tests', () => {
  it('it renders', () => {
    const missions = renderer
      .create(
        <Provider store={store}>
          <Missions />
        </Provider>,
      )
      .toJSON();
    expect(missions).toMatchSnapshot();
  });
});
