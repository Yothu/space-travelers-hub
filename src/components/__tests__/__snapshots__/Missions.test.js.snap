// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Missions page tests it renders 1`] = `
<div
  className="sc-gKclnd jZczCV"
>
  <table
    className="sc-iCfMLu jCBYFq"
  >
    <tbody>
      <tr>
        <th
          className="sc-furwcr iVKMVR font-1"
        >
          Mission
        </th>
        <th
          className="sc-furwcr iVKMVR font-1"
        >
          Description
        </th>
        <th
          className="sc-furwcr iVKMVR font-1"
        >
          Status
        </th>
        <th
          className="sc-furwcr iVKMVR font-1"
        >
           
        </th>
      </tr>
    </tbody>
  </table>
</div>
`;
